opennms-doc/guide-admin/src/asciidoc/text/compatibility.adoc

```diff
@@ -0,0 +1,17 @@
+[[compatibility]]
+== Compatibility
+
+{opennms-product-name} {opennms-version} requires the following component versions:
+
+[options="header, autowidth"]
+|===
+| Component               | Version Compatibility
+| OpenNMS Helm            | 3+
+| OpenNMS Integration API | 0.2.x
+| Cassandra               | 3.11.+
+| Elasticsearch           | 7.x
+| Java Development Kit    | OpenJDK 8, OpenJDK 11
+| Kafka                   | 2.3.0
+| PostgreSQL              | 10.x - 11.x
```

Comment:
we technically allow 10-12, only recommend 11 because 12 had come out moments before the horizon release